final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,54 @@
+#!rss_venv/bin/python
+
+"""rss-reader.py 1.0
+
+Usage:
+    rss-reader.py (-h | --help)      Show help message and exit
+    rss-reader.py <rss-url>          Print rss feeds in human-readable format
+    rss-reader.py --version          Print version info
+    rss-reader.py --json             Print result as JSON in stdout
+    rss-reader.py --verbose          Outputs verbose status messages
+    rss-reader.py --limit LIMIT      Limit news topics if this parameter provided
+"""
+
+
+def main():
+    """Main function that runs the program"""
+
+    from classes.arguments import ComLineArgParser
+    import functions.check_func as ch_f
+    import functions.process_func as proc_f
+    import functions.print_func as print_f
```

Comment:
It is better to have imports at the top of the module, not inside functions/classes

Reply:
Fixed. 
I used import inside function, because i wanted to handle all exception in one try/except block.